tests/e2e/support/objects/app-files/spaces/actions.ts

```diff
@@ -206,3 +208,45 @@ export const openSpaceTrashBin = async (args: openSpaceTrashBinArgs): Promise<vo
   await page.locator(spaceContextButton).click()
   await page.locator(spaceDeletedFilesButton).click()
 }
+
+export interface removeAccessMembersArgs {
+  users: User[]
+  page: Page
+}
+export const removeAccessSpaceMembers = async (args: removeAccessMembersArgs): Promise<void> => {
+  const { page, users } = args
+  await sidebar.open({ page: page })
+  await sidebar.openPanel({ page: page, name: 'space-share' })
+  await removeSharee({ page, users })
+}
+
+export interface searchForSpacesIdsArgs {
+  spaceID: string
+  page: Page
+}
+export const searchForSpacesIds = async (args: searchForSpacesIdsArgs): Promise<boolean> => {
```

Comment:
could you create please `e2e/support/objects/app-files/spaces/utils.ts` like  `e2e/support/objects/app-files/recource/utils.ts` and move this method there with name something like:`SpaceWithSpaceIDExist`

Reply:
sure !!